手动整理中`整理方式`增加`自动`

wikrin opened the pull request:
已包含 #252
- 仅当`目的路径`为空时，才会设为`自动`
- 已选择`整理方式`后, 再选择配置的`媒体库目录`整理方式不再改变
- `自定义路径`:
- - `整理方式`为`自动`时，会修改为`复制`,请注意
- - `整理方式`不为`自动`时, 不会改变

清空`目的路径` `整理方式`将恢复为`自动`

wikrin committed:
手动整理中`整理方式`增加`自动`

jxxghp commented on src/api/constants.ts:
```diff
@@ -73,6 +73,7 @@ export const storageDict = storageOptions.reduce((dict, item) => {
 }, {} as Record<string, string>)
 
 export const transferTypeOptions = [
+  { title: '自动', value: '' },
```
空传入后台会直接报错，没有匹配到目录的情况下

wikrin replied:
没有匹配到目录只有`自定义路径`会是空值,这种情况下在 #252 设为了`copy`, 但还是避免不了存在手动改为`自动`这种情况

wikrin committed:
去除`自动`选项
- 仅当`目的路径`为空时，才会设为`自动`
- 已选择`整理方式`后, 再选择配置的`媒体库目录`整理方式不再改变
- `自定义路径`时:
- - `整理方式`为自动时，会修改为`复制`,请注意
- - `整理方式`不为自动时, 不会改变
